app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -334,9 +335,11 @@ protected void onCreate(Bundle savedInstanceState) {
         checkForCrash();
 
         // Show the launch dialogs, if needed.
-        if (!showTermsServiceDialogIfNeeded()) {
-            if (!showPrivacyDialogIfNeeded()) {
-                showWhatsNewDialogIfNeeded();
+        if (!showDeprecatedVersionDialogIfNeeded()) {
```

Comment:
This workflow forces you to duplicate the usual flow inside that method. I think it'd be better to leave the if chain untouched and call this method unconditionally before.

Reply:
You still need to address this I think

Reply:
@svillar Updated. I looked into a more complex solution to chain the calls, but this simple one seems to work well.